src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -155,6 +166,25 @@ class BuiltinLower : public StmtExprMutator {
       return StmtExprMutator::VisitExpr_(op);
     }
   }
+  std::string GetUniqueName(std::string prefix) {
+    for (size_t i = 0; i < prefix.size(); ++i) {
+      if (prefix[i] == '.') prefix[i] = '_';
+    }
```

Comment:
how about use STL style code "std::replace(prefix.begin(), prefix.end(), '.', '-');"  to simple this part logic?

Reply:
The idea to use "std::replace" is perfect. But just to clarify, the whole "GetUniqueName()" function is copied from https://github.com/apache/tvm/blob/main/src/target/source/codegen_source_base.cc#L35. I searched over the whole tvm codebase and I found there are a lot components implement the "GetUniqueName()" function, which have pretty much the same functionality. What about we could make the "GetUniqueName()" a utility function shared by all the components, to avoid the duplication?

Reply:
@zhuwenxi, thanks for the clarify, the idea make GetUniqueName to a utility function is a great idea, I also understand this code style is existing issue, either keep it on this patch or make a utility function and fix is ok to me.